src/styles/components/form-elements/form-row/styles.less

```diff
@@ -14,6 +14,11 @@
       margin-bottom: 0;
     }
   }
+
+  // Add top-padding equivalent to a label
+  .form-group-without-top-label {
+    padding-top: 1.82rem;
+  }
```

Comment:
Ah, this makes more sense (I didn't much like the magic number either). 

Thanks for the very clear explanation and references! 🙌